ansible/roles/kong-api/defaults/main.yml

```diff
@@ -5692,3 +5693,1101 @@ kong_apis:
       config.limit_by: credential
     - name: request-size-limiting
       config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: tags
```

Comment:
please give a meaningful name for this API!

Reply:
discussionForumTags